h2o-docs/src/product/data-science/deep-learning.rst

```diff
@@ -217,7 +217,7 @@ H2O Deep Learning models have many input parameters, many of which are only acce
 
 -  **force_load_balance**: Specify whether to force extra load balancing to increase training speed for small datasets and use all cores. This option is defaults to true (enabled).
 
--  **variable_importances**: Specify whether to compute variable importance. This option is defaults to false (not enabled).
+-  **variable_importances**: Specify whether to compute variable importance. This option is defaults to true.
```

Comment:
Defaults to `true` for the last 4 years. https://github.com/h2oai/h2o-3/blob/03c73caf76c4eb4069e70490140435f0bc64f79d/h2o-algos/src/main/java/hex/deeplearning/DeepLearningModel.java#L1617